### PR TITLE
bug/update validator

### PR DESCRIPTION
**Pull request by borovHokage**

### CODE REVIEW

File: src/NewsService.Mappers/Models/NewsMapper.cs [hunk 1]
```diff
@@ -19,7 +20,7 @@ public DbNews Map(News request)
                 Id = Guid.NewGuid(),
                 Content = request.Content,
                 Subject = request.Subject,
-                Pseudonym = request.Pseudonym,
+                Pseudonym = request.Pseudonym.Trim().Any() ? request.Pseudonym.Trim() : null,
```
Comment (BelehovEgor) [inline src/NewsService.Mappers/Models/NewsMapper.cs:23]: if pseudonym is null, you will get null ref exc

**Review by BelehovEgor — CHANGES_REQUESTED** [review]

**Review by BelehovEgor — APPROVED** [review]

File: src/NewsService.Mappers/Models/NewsMapper.cs (hunk 1 above)
Comment (borovHokage) [reply src/NewsService.Mappers/Models/NewsMapper.cs]: fixed

**Review by MariyaSin — APPROVED** [review]

File: Common.Configurations.targets [hunk 2]
```diff
@@ -33,8 +33,8 @@
   </ItemGroup>
 
   <ItemGroup Condition="$(AssemblyName.Contains('.UnitTests')) != 'true'">
-    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.27" />
-    <PackageReference Include="LT.DigitalOffice.Models" Version="1.0.6" />
+    <PackageReference Include="LT.DigitalOffice.Kernel" Version="1.1.29" />
```
Comment (TIove) [inline Common.Configurations.targets:36]: Change to 1.1.32

Comment (borovHokage) [reply Common.Configurations.targets]: done

**Review by djino97 — APPROVED** [review]

**Pull request merged**